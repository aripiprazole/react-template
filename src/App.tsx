import React from "react";

import { Routes } from "~/routes/Routes";

import { GlobalStyle } from "~/styles";

const App: React.FC = () => (
  <>
    <Routes />

    <GlobalStyle />
  </>
);

export default App;
