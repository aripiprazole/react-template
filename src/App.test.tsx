import React from "react";

import App from "~/App";

import { shallow } from "enzyme";

describe("App tests", () => {
  it("should render correctly", () => {
    const tree = shallow(<App />).html();

    expect(tree).toMatchSnapshot();
  });
});
