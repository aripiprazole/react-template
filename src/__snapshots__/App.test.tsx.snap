// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render correctly 1`] = `"<div class=\\"sc-AxjAm iSRPDj\\"><header class=\\"sc-AxiKw nENPn\\"><img src=\\"logo.svg\\" class=\\"sc-AxhCb fiKusx App-logo\\" alt=\\"logo\\"/><p>Edit <code class=\\"sc-AxirZ fCPBbK\\">src/App.tsx</code> and save to reload.</p><a class=\\"App-link\\" href=\\"https://reactjs.org\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Learn React</a></header></div>"`;
